wpilibNewCommands/src/main/java/edu/wpi/first/wpilibj2/command/Command.java

```diff
@@ -285,13 +285,34 @@ public SequentialCommandGroup andThen(Command... next) {
    * commands with {@link CommandScheduler#removeComposedCommand(Command)}. The command composition
    * returned from this method can be further decorated without issue.
    *
-   * @param parallel the commands to run in parallel
+   * @param parallel the commands to run in parallel. Note the parallel commands will be interupted
+   *     when the original command ends
```

Comment:
This change should be on the new method, not the deprecated one. And see the `original->deadline` reword comment on C++